src/router.js

```diff
@@ -80,15 +80,14 @@ export class Router {
   * @param viewPort The viewPort.
   * @param name The name of the viewPort. 'default' if unspecified.
   */
-  registerViewPort(viewPort:Object, name?:string) {
-    name = name || 'default';
+  registerViewPort(viewPort : Object, name? : string = 'default') : void {
```

Comment:
Well, I checked the behavior in the above fiddle and it doesn't fail. If an empty string is passed, it doesn't use the `default`. So I don't see why it'll fail. Is there any test that cover this, or perhaps I should add it...